### PR TITLE
correct router problem

### PR DESCRIPTION
**Pull request by hoslack**

### CODE REVIEW

**Review by MarlenAw — APPROVED** [review]

**Comment by MynahMarie** [comment]
@hoslack I tried to pull your branch and try it and it still says that router is undefined

**Comment by Karyum** [comment]
@hoslack have no idea why those changes were, keep the auth router the way it is , and just change 
https://github.com/FACN3/easyRep/blob/master/server/routes/reportRoutes.js#L6 this to be `reportRouter.post`

**Comment by Karyum** [comment]
@hoslack **also don't skip your teammates review next time**.

**Comment by hoslack** [comment]
@MynahMarie Please send us the facebook local env variables to test the app

**Comment by MynahMarie** [comment]
@hoslack Sent. Please send me the variables for the email sending also.

**Comment by Karyum** [comment]
@hoslack @MynahMarie can i merge it now? i need the confirmation from both of you

File: server/routes/authRoutes.js [hunk 1]
```diff
@@ -1,34 +1,27 @@
 const authRouter = require('express').Router();
+const router = require('express').Router();
```
Comment (Karyum) [inline server/routes/authRoutes.js]: duplicated line here

File: server/routes/reportRoutes.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
-const reportRouter = require('express').Router();
+const router = require('express').Router();
```
Comment (Karyum) [inline server/routes/reportRoutes.js]: reportRouter for readability

File: server/routes/reportRoutes.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 const createReport = require('../controllers/createReport');
 const sendEmail = require('../controllers/sendEmail');
 
 //add report to the database
 router.post('/create_report', createReport);
```
Comment (Karyum) [inline server/routes/reportRoutes.js]: :arrow_up:

File: server/routes/reportRoutes.js [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
 router.post('/send_email', sendEmail);
 
-module.exports = reportRouter;
+module.exports = router;
```
Comment (Karyum) [inline server/routes/reportRoutes.js]: :arrow_up:

**Review by Karyum — CHANGES_REQUESTED** [review]

**Comment by hoslack** [comment]
@MynahMarie Would you mind coming to class so that we work as a team?

File: server/routes/reportRoutes.js (hunk 2 above)
Comment (hoslack) [reply server/routes/reportRoutes.js]: Done!

**Comment by MynahMarie** [comment]
Ok I think this is ready to be merged

**Pull request merged**